config-templates/redhat/ocs-remote/4.7/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "4.7",
+  "enabled": "true",
+  "description": "OCS template for openshift 4.7 (Remote storage)",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "redhat"
+    },
+    {
+      "name": "provider-logo",
+      "value": "logo/ocs-remote.svg"
+    },
+    {
+      "name": "kube-version",
```

Comment:
Update key `kube-version` to `supported-k8s-versions`
Provide comma separated list of supported Kube versions like `"1.18,1.19"`